src/pages/administerUsersPage.vue

```diff
@@ -1,39 +1,280 @@
 <template>
```

Comment:
Vi sal nok være lidt obs på v-model navne hvis vi skal have flere/nogle af vores dialogs ind i templates. Ikke sikker på at det er relevant her, nu når det her kun er for brugere der er logget ind og notificationDialog ikke bruger noget tilsvarende, bare lige en tanke